Assets/Content/Creatures/Creature.cs

```diff
@@ -5,10 +5,17 @@
 using UnityEngine;
 using UnityEngine.XR;
 
+// TODO: First and foremost, I should rename this to Entity.
+
+// An entity should be used for anything that ressembles the creature idea
+// it can walk, interact with stuff, be controlled by a player
```

Comment:
Would this include ghosts too? If so, I'd just put "controlled by player" and remove the walking, interacting, & combat.

Reply:
Correct, a ghost, while not visible or interactable, is controlled by the player and can interact with the map and entities (you can scare people and flick lights)